server/index.js

```diff
@@ -24,79 +23,93 @@ app.use('/', expressStaticGzip(path.join(__dirname, '../client'), {
 
 app.use(cors());
 
-// ----------------- CRUD API REQUESTS ------------------ //
+// ---------------- CRUD FOR POSTGRESQL --------------- //
 
-app.get('/songdata/:id', async (req, res) => {
-  try {
-    var id = req.params.id;
-    if (id > 100 || id < 0) {
-      throw new Error(`songID: ${id} does not exist`);
-    }
-    var song = await Song.findOne({ songID: id });
-    res.status(200).json({
-      status: "Success",
-      data: song
-    });
-  } catch (error) {
-    console.log(error)
-  }
-});
-
-// app.get('/songsByBandID/:id', async (req, res) => {
-//   try {
-//     var bandID = req.params.id;
-//     if (bandID > 30 || bandID < 0) {
-//       res.end('BAND ID DOES NOT EXIST');
-//     } else {
-//       var findSongsByBand = await songData.findSongsByBand(bandID)
-//       res.send(findSongsByBand);
-//     }
-//   } catch (error) {
-//     console.log(error);
-//   }
-// });
+// CREATE
 
 app.post('/song', async (req, res) => {
   try {
-    var song = await Song.create(req.body);
+    var song = await knex('song_list').insert(req.body);
+    // console.log('CREATE SONG request data: ', req.body);
     res.status(201).json({
       status: 'Success',
-      data: song
+      data: req.body
     });
   } catch (error) {
-    console.log('Error creating new song: ', error);
+    console.log(chalk.red('Error creating new song: ', error));
     res.status(400).json({
       status: 'Failed',
       data: error
     });
   }
 });
 
+// READ
+
+app.get('/songdata/:id', async (req, res) => {
+  try {
+    var id = req.params.id;
+    if (id > 10000000 || id < 0) {
```

Comment:
What happens if an `id` is not a number here? I think what will happen is that it will send a response with a status of 200 per line 62 because `NaN > 10000000` and `NaN < 0` returns `false` for both. Consider this wherever you're verifying your `id` like so.

Reply:
Done!